docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  php:
+    build:
+      context: ./php
+      dockerfile: php.Dockerfile
```

Comment:
what is this php.Dockerfile? Can you upload it too? @Gui-go

Reply:
File php.Dockerfile [added](https://github.com/geomunduswebteam/geomundus2024/blob/guilherme/php.Dockerfile) in the branch "Guilherme".
Please note that the docker-compose.yml file should be in the parent directory to run the service.